src/app/_index.js

```diff
@@ -2,6 +2,7 @@ import path from 'path'
 import { store } from 'store'
 import { ipcRenderer } from 'electron'
 import { dialog, getCurrentWindow } from '@electron/remote'
+import { rm } from 'fs'
```

Comment:
do we need this anymore?

Reply:
You're right.  Good catch.  It should fail when we export via the file menu at the moment.
I'll double-check that that's the case for sanity's sake.